### ISSUE TITLE
"cloudcode.autoDependencies": "off" -> "Cloud SDK not found. Install Cloud SDK manually following these steps."

### ISSUE THREAD

**Issue by vindicatorr** [GoogleCloudPlatform/cloud-code-vscode#591]
Issue Type: <b>Bug</b>

"Cloud SDK not found. Install Cloud SDK manually following these steps."

When "cloudcode.autoDependencies": "off"
- Managed Dependencies: off
- Cloud SDK Version: Not installed
- Skaffold Version: Not installed
- Minikube Version: Not installed

When "cloudcode.autoDependencies": "on"
- Managed Dependencies: on
- Cloud SDK Version: 387.0.0
- Skaffold Version: v1.38.0
- Minikube Version: 1.25.2

Extension version: 1.19.0-insiders.0
VS Code version: Code - OSS 1.67.0 (e6d98a13fcd3bf928d0e755eaae873ac200a98bd, 2022-04-18T02:52:33.790Z)
OS version: Linux x64 5.18.0-gfdaf9a5840ac-dirty
Restricted Mode: No

```
$ echo $PATH
/home/userName/.cache/cloud-code/installer/google-cloud-sdk/bin:<...>

$ ls /home/userName/.cache/cloud-code/installer/google-cloud-sdk/bin
anthoscli      dev_appserver.py          gcloud                    gsutil                 kubectl.1.18  kubectl.1.21  minikube
bootstrapping  docker-credential-gcloud  git-credential-gcloud.sh  java_dev_appserver.sh  kubectl.1.19  kubectl.1.22  skaffold
bq             endpointscfg.py           gke-gcloud-auth-plugin    kubectl                kubectl.1.20  kubectl.1.23
```

The gcloud meta list-commands ends up taking ~1GB before finishing.
I'm looking to disable the "autoDependencies" so I can forego that step, and just re-enable it when I have free-time/RAM every now and again.
Basically do a manual check. It'd be good if it just checks for updates and lets the user perform the update now or later.
